copy-this-template-directory/index.md

```diff
@@ -1,16 +1,1050 @@
+                        label: {
+                            Text("Register")
+                                .font(.title)
+                                .foregroundColor(.white)
+                                .padding()
+                                .background(Color.orange)
+                                .cornerRadius(10)
+                        })
+                    .padding()
+                }
+                
+                Spacer()
+            }
+            .padding()
+            .navigationTitle("Welcome")
+        }
+    }
+    
+    func logout() {
+        guard let refreshToken = accessToken
+        else {
+            print("Refresh token is missing")
+            return
+        }
+        
+        guard let url = URL(string: "https://directus.lws.io/auth/logout") else {
+            print("Invalid logout URL")
+            return
+        }
+        
+        var request = URLRequest(url: url)
+        request.httpMethod = "POST"
+        request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+        
+        let body: [String: Any] = [
+            "refresh_token": refreshToken
+        ]
+        
+        do {
+            request.httpBody = try JSONSerialization.data(withJSONObject: body)
+        } catch {
+            print("Error encoding request body: \(error.localizedDescription)")
+            return
+        }
+        
+        URLSession.shared.dataTask(with: request) { data, response, error in
+            if let error = error {
+                print("Error logging out: \(error.localizedDescription)")
+                return
+            }
+            
+            if let httpResponse = response as? HTTPURLResponse {
+                if (200..<300).contains(httpResponse.statusCode) {
+
+                    DispatchQueue.main.async {
+                        isLoggedIn = false
+                        accessToken = nil
+                    }
+                } else {
+                    print("Failed to logout. Status code: \(httpResponse.statusCode)")
+                }
+            }
+        }.resume()
+    }
+    
+    struct ContentView_Previews: PreviewProvider {
+        static var previews: some View {
+            ContentView()
+        }
+    }
+}
+
+```
+### ContentView Struct
+
+Defines the main view of the application.
+
+### State Properties:
+
+Three `@State` properties are declared to manage the state of the view:
+
+- **showLoginView**: Tracks whether the login view should be displayed.
+- **isLoggedIn**: Tracks whether the user is logged in.
+- **accessToken**: Stores the access token after successful login.
+
+### Body Property:
+
+The `body` property defines the view's layout and behavior.
+
+- **NavigationView**: Wraps the content and provides navigation functionality.
+- **VStack**: Vertical stack layout for arranging child views.
+- **Spacer**: Flexible space to push content to the top and bottom.
```

Comment:
Irrelevant. You don't need to explain every line - just those important to the functionality.